src/features/locators/styles/locatorEditDialog.less

```diff
@@ -1,9 +1,21 @@
 @import "../../../common/styles/variables.less";
 
 .jdn__locatorEdit-warning {
-    margin-top: 8px;
+  margin-top: 8px;
 
-    & svg {
-        color: @warning-icon-color;
-    }
-}
\ No newline at end of file
+  & svg {
+    color: @warning-icon-color;
+  }
+}
+
+.ant-form-item:last-child {
+  margin-bottom: 0;
+}
+
+.ant-form {
+  min-height: 238px;
+}
+
+.ant-modal-body {
+  padding-bottom: 12px;
+}

```

Comment:
please, incapsulate these styles inside .jdn__locatorEdit, for not tot allow them affect other forms and modals in app